components/bl-AnimateOnScroll-component/README.md

```diff
@@ -0,0 +1,27 @@
+# Animate on Scroll
+
+Animate on Scroll is a component of Backendless UI-Builder designer. This component allows adding animation for elements when scrolling the page.
+
+The component based on external [Animate on Scroll](https://github.com/michalsnik/aos).
+
+## Properties
+
+| Property       | Type                                                                                                                                                                                                                                                                                                                                           | Default Value | Logic         | Data Binding | UI Setting | Description                                                                                                       |
+|----------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------|---------------|--------------|------------|-------------------------------------------------------------------------------------------------------------------|
+| Disable        | Select ["mobile", "tablet", "true", "false"]                                                                                                                                                                                                                                                                                                   | "false"       | Disable Logic | YES          | YES        | Allows to determine whether or not to disable the animation                                                       |
```

Comment:
```suggestion
| Disable        | *Select* <br/>[`mobile`, `tablet`, `true`, `false`]                                                                                                                                                                                                                                                                                                   | "false"       | Disable Logic | YES          | YES        | Allows to determine whether or not to disable the animation                                                       |
```
We make select in this way [Example](https://github.com/Backendless/ui-builder-library/blob/bl-AnimateOnScroll-component/components/bl-leafletMap-component/README.md#properties)

Reply:
Also all types are in italic style ` *Number* `